website/src/data/tweets.tsx

```diff
@@ -37,7 +37,7 @@ const TWEETS: TweetItem[] = [
     name: 'Maël',
     date: 'Jan 20, 2021',
     avatar:
-      'https://pbs.twimg.com/profile_images/1311259425949261825/7hPZqoJd_400x400.jpg',
+      'https://pbs.twimg.com/profile_images/1497225853931040769/5mXZAHFR_400x400.jpg',
```

Comment:
<img width="446" alt="image" src="https://user-images.githubusercontent.com/55398995/161965287-113abaca-ec8b-4c7d-8bb9-ae6d125e98c9.png">

Is it just me or is swyx's avatar suddently becoming 404?